test/input/beamville/beam.conf

```diff
@@ -110,6 +110,11 @@ beam.router.skim = {
     fileBaseName = "skimsTAZ"
   }
 }
+#helics
+beam.cosim.helics = {
```

Comment:
Should it be added?

Reply:
Yes I keep it. 
Enabling of ChargingNetworkManager is configured by a separate property